lib/jsdom/level2/events.js

```diff
@@ -157,6 +157,27 @@ inheritFrom(core.Event, core.MutationEvent, {
     REMOVAL      : 3
 });
 
+core.ProgressEvent = function(eventType, eventInit) {
```

Comment:
For this, can you follow the pattern used for https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/living/hash-change-event.js, and place it in its own file, and so on? (You can also use class syntax if you like, or even webidl2js, but at least following that pattern would be helpful, and would ensure that things like getters vs. instance properties are correct.)